src/main/resources/assets/admin/common/js/ui/selector/Option.ts

```diff
@@ -1,5 +1,82 @@
-export interface Option<T>
-    extends Slick.SlickData {
+import {IDentifiable} from '../../IDentifiable';
+
+export class Option<T>
+    implements Slick.SlickData, IDentifiable {
+
+    private value: string;
+
+    private displayValue: T;
+
+    private indices?: string[];
+
+    private readOnly?: boolean;
+
+    private empty?: boolean;
+
+    private selectable?: boolean;
+
+    private expandable?: boolean;
+
+    constructor(builder: OptionBuilder<T>) {
```

Comment:
Codacy found an issue: ['OptionBuilder' was used before it was defined.](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6401650)